src/FlashList.tsx

```diff
@@ -350,6 +351,7 @@ class FlashList<T> extends React.PureComponent<
           windowCorrectionConfig={this.getUpdatedWindowCorrectionConfig()}
           itemAnimator={this.itemAnimator}
           suppressBoundedSizeException
+          externalScrollView={renderScrollComponent as any}
```

Comment:
Typecast to BaseScrollView instead

Reply:
i had to cast it to `RecyclerListViewProps["externalScrollView"]` instead

```ts
    externalScrollView?: {
        new (props: ScrollViewDefaultProps): BaseScrollView;
    };
```